app/helpers/__init__.py

```diff
@@ -0,0 +1,88 @@
+import time
+
+import boto3.exceptions as boto3_exc
+from boto3.dynamodb.conditions import Key
+from flask import abort
+
+from app.helpers.checks import check_and_get_url_short
+from app.helpers.response_generation import make_error_msg
+from app.models.dynamo_db import get_dynamodb_table
+from app import app
+config = app.config
+
+
+def create_url(table, url):
+    """
+    This function creates an id which will act as the shortened url and returns it
+
+    raise Exceptions when there is an issue during the dynamoo db table write
+
+    :param table: the aws table on which we will write
+    :param url: the url we want to shorten
+    :return: the shortened url id
+    """
+    try:
+        # we create a magic number based on epoch for our shortened_url id
+        # urls have a maximum size of 2046 character due to a dynamodb limitation
+        if len(url) > 2046:
+            return "toolong", \
+                   f"The url given as parameter was too long. (limit is 2046 characters, {len(url)} given)"
+        t = int(time.time() * 1000) - 1000000000000
+        shortened_url = '%x' % t
+        now = time.localtime()
+        table.put_item(
+            Item={
+                'url_short': shortened_url,
+                'url': url,
+                'timestamp': time.strftime('%Y-%m-%d %X', now),
+                'epoch': time.strftime('%s', now)
+            }
+        )
+        return shortened_url, None
+    # Those are internal server error: error code 500
+    except boto3_exc.Boto3Error as e:
+        abort(make_error_msg(500, f"Write units exceeded: {str(e)}"))
+
+
+def fetch_url(url_id):
```

Comment:
Some functions (e.g. create_url just above) are pretty nicely commented, some others (like this one) aren't..
Perhaps some functions are pretty self-explaining, but maybe some short more or less default comments on the parameters, the returns and the in-between would be nice.